app/git/utils.py

```diff
@@ -475,6 +475,9 @@ def get_interested_actions(github_url, username, email=''):
         elif 'user' in action:
             gh_user = action['user']['login']
 
+        if gh_user and gh_user == username and action['event'] in activity_event_types:
+            actions_by_interested_party.append(action)
+
         if action['event'] == 'cross-referenced':
```

Comment:
Actually, we shouldn't fetch further at this point.
It's likely that we fetch things and (prs) from other issues, that gives us false-positives 🤔

Reply:
wouldnt fetching more things / prs from other issues give us less false-negatives (more PRs, more chances we could find a place where the contributor commented)

Reply:
That was the bug I'm talking about first (variable assignment/overwrite) - the gh_user.

With false-negatives I mean if we fetch further we very likely fetch PRs unrelated to the issue in question.
If we just do one loop we should get all the events we need, at least if the user references the issue in the commits/pull request(s). That may also include false-negatives, but that would be rare.